packages/@angular/cli/commands/update.ts

```diff
@@ -0,0 +1,48 @@
+const Command = require('../ember-cli/lib/models/command');
+import { UpdateTask } from '../tasks/update';
+
+export interface UpdateOptions {
+  schematic?: boolean;
+}
+
+const UpdateCommand = Command.extend({
+  name: 'update',
+  description: 'Opens the official Angular API documentation for a given keyword.',
+  works: 'everywhere',
+  availableOptions: [
+    {
+      name: 'dry-run',
+      type: Boolean,
+      default: false,
+      aliases: ['d'],
+      description: 'Run through without making any changes.'
+    },
+    {
+      name: 'force',
+      type: Boolean,
+      default: false,
+      aliases: ['f'],
+      description: 'Forces overwriting of files.'
+    }
+  ],
+
+  anonymousOptions: [
+    '<schematic>'
+  ],
+
+  run: function(commandOptions: any, rawArgs: Array<string>) {
+    let schematic = rawArgs[0] || '@schematics/package-update:all';
+    if (!schematic.match(/:/)) {
+      schematic = `@schematics/package-update:${schematic}`;
```

Comment:
should this be restricted to just the `package-update` collection?  maybe even validated against the list of schematics in the collection as well?  Not sure the enduser should even be aware schematics are even in use.

Reply:
It is not limited to just `@schematics/package-update` but if you want to use another schematic lib, you need to specify the entire schematic name (`collectionName:schematicName`). As for the validation, that error will be handled/thrown by the schematics engine.